.github/workflows/haskell.yml

```diff
@@ -6,24 +6,24 @@ on:
   pull_request:
     branches: [ 'master' ]
 
-env:
-  autoconf_ver: 2.69
+# env:
+#   autoconf_ver: 2.69
 
 jobs:
   autoconf:
     runs-on: ubuntu-latest
-    container: ubuntu:22.10
+    # container: ubuntu:22.10
```

Comment:
We could remove that line altogether.
If it is important that this CI run in a certain container, that should be justified here in a comment.